wilcard/src/main/java/org/fundacionjala/Wilcard/SearchEngine.java

```diff
@@ -0,0 +1,61 @@
+package org.fundacionjala.Wilcard;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by erickaviraca on 9/2/2016.
+ */
+public class SearchEngine {
+
+    /**
+     * Finds and return the position of the pattern matches.
+     *
+     * @param needle, the original needle.
+     * @param haystack, the string when the needle will evaluate.
+     * @return --> the position (an integer number). and if it isn't found (-1)
+     */
+    public static int find(String needle, String haystack) {
```

Comment:
@ErickaViraca 
You have a lot of code.
Review again the Pattern and Matcher classes.

You can have the solution with 3 lines of code. :)